Configuration/AlCa/python/autoCond.py

```diff
@@ -52,13 +54,15 @@
     # GlobalTag for MC production with perfectly aligned and calibrated detector for full Phase1 2018 (and 0,0,0-centred beamspot)
     'phase1_2018_design'       : '103X_upgrade2018_design_v4',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector
-    'phase1_2018_realistic'    : '103X_upgrade2018_realistic_v7',
+    'phase1_2018_realistic'    : '103X_upgrade2018_realistic_v8',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector for Heavy Ion
-    'phase1_2018_realistic_hi' : '103X_upgrade2018_realistic_HI_v7',
+    'phase1_2018_realistic_hi' : '103X_upgrade2018_realistic_HI_v9',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector: HEM-15-16 fail
-    'phase1_2018_realistic_HEfail'    : '103X_upgrade2018_realistic_HEfail_v3',
+    'phase1_2018_realistic_HEfail'    : '103X_upgrade2018_realistic_HEfail_v4',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in DECO mode
-    'phase1_2018_cosmics'      :   '103X_upgrade2018cosmics_realistic_deco_v6',
+    'phase1_2018_cosmics'      :   '103X_upgrade2018cosmics_realistic_deco_v7',
+    # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
+    'phase1_2018_cosmics'      :   '103X_upgrade2018cosmics_realistic_peak_v1',
```

Comment:
Hi @lpernie , I guess you meant phase1_2018_cosmics_peak for this one (otherwise you have a duplication in the dictionnary)  Thanks

Reply:
[My comment can be also applied to other version (master etc..) of this PR ]

Reply:
Correct, fixed now!